config/crds/verify_v1beta1_certificaterequest.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   names:
     kind: CertificateRequest
     plural: certificaterequests
-  scope: Cluster
+  scope: Namespaced
```

Comment:
Is that what we want it to be on the bigger scale?

Reply:
Great question! and I'm not sure I have a good answer...

We wanted it to be Cluster scoped initially since that totally makes sense for a RootCA... however we also want intermediate certs that are namespaced (so RBAC can be used to control the owner)...

I felt introducing two CustomResources at once that are basically identical - ClusterCertificateRequest and CertificateRequest was probably overkill for this first attempt ... but I think we may want to revisit this.